source/docs/partnering/definitions/expert.yml

```diff
@@ -1,77 +1,78 @@
-expert:
-  type: object
-  properties:
-    uuid:
-      type: string
-      example: dc56d205-4ddf-46d4-bcd6-93ddc7fea84a
-    first_name:
-      type: string
-      example: Daniel
-    last_name:
-      type: string
-      example: Abraham
-    lab:
-      type: string
-      example: Argonne National Laboratory
-    avatar:
-      type: string
-      example: https://www.labpartnering.org/experts/dc56d205-4ddf-46d4-bcd6-93ddc7fea84a/avatar
-    links:
-      type: object
-      properties:
-        self:
-          type: string
-          example: https://developer.nrel.gov/api/lps/v1/experts/dc56d205-4ddf-46d4-bcd6-93ddc7fea84a
-    bio:
-      type: string
-      example: He is well known for his expertise in the field of lithium batteries at Argonne National Laboratory. Since graduating with a doctorate in metallurgical engineering from the University of Illinois at Urbana-Champaign
-#    specialties:
-#      type: array
-#    technologies:
-#      type: array
-  required:
-    - uuid
-    - first_name
-    - last_name
-    - lab
-    - bio
-experts:
-  type: array
-  items:
-    $ref: "#/expert"
-experts_response:
-  type: object
-  properties:
-    metadata:
-      type: object
-      $ref: "./metadata.yml#/metadata"
-#    inputs:
+# This file appears to be ignored
+#expert:
+#  type: object
+#  properties:
+#    uuid:
+#      type: string
+#      example: dc56d205-4ddf-46d4-bcd6-93ddc7fea84a
+#    first_name:
+#      type: string
+#      example: Daniel
+#    last_name:
+#      type: string
+#      example: Abraham
+#    lab:
+#      type: string
+#      example: Argonne National Laboratory
+#    avatar:
+#      type: string
+#      example: https://www.labpartnering.org/experts/dc56d205-4ddf-46d4-bcd6-93ddc7fea84a/avatar
+#    links:
```

Comment:
Can all of these commented out files be deleted altogether? That might just be less confusing in the future.

Reply:
Sure, done.